src/custom/state/orders/middleware.ts

```diff
@@ -224,12 +224,11 @@ export const soundMiddleware: Middleware<Record<string, unknown>, AppState> = (s
 }
 
 export const appziMiddleware: Middleware<Record<string, unknown>, AppState> = (store) => (next) => (action) => {
-  const result = next(action)
-
-  if (isBatchFulfillOrderAction(action) || isSingleFulfillOrderAction(action)) {
+  if (isBatchFulfillOrderAction(action) || isSingleFulfillOrderAction(action) || isExpireOrdersAction(action)) {
```

Comment:
This will be called two times for orders which were open for more than 5min

I guess theres no problem with that because appzi won't trigger it twice (because of the `Sometimes` :) ) right?

Reply:
Yes, the display of the modal is controlled by Appzi, so it won't repeat if it doesn't have to